File: src/TmrOverlay.App/Overlays/InputState/InputStateOverlayDefinition.cs
using TmrOverlay.Core.Overlays;

namespace TmrOverlay.App.Overlays.InputState;

internal static class InputStateOverlayDefinition
{
    public static OverlayDefinition Definition { get; } = new(
        Id: "input-state",
        DisplayName: "Inputs",
        DefaultWidth: 520,
        DefaultHeight: 260,
        FadeWhenLiveTelemetryUnavailable: true,
        ContextRequirement: OverlayContextRequirement.LocalPlayerInCar);
}

File: local-mac/TmrOverlayMac/Sources/TmrOverlayMac/Overlays/InputState/InputStateOverlayDefinition.swift
import AppKit

enum InputStateOverlayDefinition {
    static let definition = OverlayDefinition(
        id: "input-state",
        displayName: "Inputs",
        defaultSize: NSSize(width: 520, height: 260),
        fadeWhenLiveTelemetryUnavailable: true
    )
}
